### PR TITLE
support k8s.gcr.io and envrionmental http proxy

### PR DESCRIPTION
**Pull request by Freezind**

### CODE REVIEW

**Comment by marshallford** [comment]
Any updates on this?

**Comment by josegonzalez** [comment]
Mind rebasing this onto master so we can trigger builds?

File: mirror.go [hunk 1]
```diff
@@ -22,10 +22,12 @@ const (
 	dockerHub = "hub.docker.com"
 	quay      = "quay.io"
 	gcr       = "gcr.io"
+	k8s_gcr   = "k8s.gcr.io"
```
Comment (kirooshu) [inline mirror.go:25]: Can you please follow the golang standard and use camel case for this variable, more or less how `dockerHub` is not `docker_hub` :) Or just call it `k8s` as this is the defactory `k8s` repo.

Comment (Freezind) [reply mirror.go]: updated, thanks for the comment.

**Comment by Freezind** [comment]
> Mind rebasing this onto master so we can trigger builds?

Done.

**Comment by josegonzalez** [comment]
@Freezind mind running `go fmt`? Seems there might be an issue in your PR.

**Comment by Freezind** [comment]
> @Freezind mind running `go fmt`? Seems there might be an issue in your PR.

Done. Sorry for that I am not familiar with go.

**Review by kirooshu — APPROVED** [review]

**Comment by Freezind** [comment]
Hi @kirooshu, can you help me merge this PR?

**Pull request merged**